authentication.css

```diff
@@ -0,0 +1,138 @@
+.authentication{
+    display: grid;
+    grid-template-columns: 50% 50%;
+    background-color: var(--background-color-dark1);
+    border-radius: 8px;
+    width: 90%;
+    max-width: 93.75rem;
+    margin: 0rem auto;
+    padding: 5rem 1rem;
+    box-shadow: 0px 3px 8px var(--tertiary-color);
+    gap: 1rem;
+}
+.wrapper{
+    position: relative;
+}
+.blob{
+    position: absolute;
+    width: 30%;
+    z-index: -1;
+}
+.blob1{
+    top: -10px;
+    left: -100px;
+}
+.blob2{
+    bottom: 100px;
```

Comment:
use rems for sizing if it is more than 10px

Reply:
It is not for sizing it is for placement, and if i will use rems it would depend on fontsize